src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/inventory/inventory.component.ts

```diff
@@ -69,6 +69,11 @@ export class InventoryComponent implements OnChanges, OnInit {
         name: this.i18n('Model'),
         prop: 'model',
         flexGrow: 1
+      },
+      {
+        name: this.i18n('OSD ID'),
+        prop: 'osd_id',
+        flexGrow: 1
```

Comment:
Can we make this a link to the osd page?

Reply:
Not at the moment, the datatable does not provide that.